app/views/course/leaderboards/show.html.slim

```diff
@@ -1,4 +1,4 @@
-= page_header
+= page_header(@settings.title)
```

Comment:
I think you need to check for `nil`, use announcement index page as a reference ?